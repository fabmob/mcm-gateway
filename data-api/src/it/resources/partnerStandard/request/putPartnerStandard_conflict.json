{
  "standardName": "FNMS update",
  "isActive": false,
  "versionStandard": "V1.1",
  "versionDataMapping": "V1.0",
  "partnerId": "6c317901-749f-4d7d-82bb-afdc737b26b3",
  "partnerActionsId": "7c8f8780-a6fa-495d-8b36-24a20539adca",
  "adaptersId": "40142f60-9694-479f-a6cd-28b199b5e240"

}