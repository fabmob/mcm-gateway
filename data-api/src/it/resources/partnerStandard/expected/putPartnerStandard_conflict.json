{
  "status": 409,
  "errorCode": 409,
  "label": "Conflict",
  "description": "The MSP with id 6c317901-749f-4d7d-82bb-afdc737b26b3 does not handle this feature : PING",
  "timestamp": "2022-10-05T15:11:11.777+02:00",
  "callId": "8690391b-ae0a-4b08-9811-c5239e848bb8"
}